### PR TITLE
InteroperabilityTest_Engine: Align to changes in BH.oM.Diffing.DiffConfig

### LINKED ISSUE #190: InteroperabilityTest_Engine: align to changes in Diffing_Engine
I didn't realise there was an usage of the diffing engine here.

In the PR of the Diffing_oM I'm [removing the two `bool` properties from the DiffConfig](https://github.com/BHoM/BHoM/pull/715/files#diff-f38ceaaaf427a8b791723bcc876d5177L62-L64) as I deemed them redundant, as they duplicate the information contained in the `PropertiesToIgnore` field. I thought I could keep them but that would require some checking that the `PropertiesToIgnore` gets updated with them in parallel, which is currently not happening, and I'd avoid such a complication. 

So in this file: 

https://github.com/BHoM/Test_Toolkit/blob/master/InteroperabilityTest_Engine/Query/IsEqual.cs

there are changes required. 

Or, if a relevant use cases arises, I can restore those properties in the Diffing, but that would need discussion. @IsakNaslundBh

### PR DESCRIPTION
**Pull request by alelom**
### NOTE: Depends on 

https://github.com/BHoM/BHoM/pull/715
https://github.com/BHoM/BHoM_Engine/pull/1534

 ### Issues addressed by this PR

 Closes #190

### CODE REVIEW

File: InteroperabilityTest_Engine/Query/IsEqual.cs [hunk 1]
```diff
@@ -58,11 +58,9 @@ public static partial class Query
 
             comparer.Config.MembersToIgnore = config.PropertiesToIgnore;
 
-            if (config.IgnoreCustomData)
-                comparer.Config.MembersToIgnore.Add("CustomData");
+            comparer.Config.MembersToIgnore.Add("CustomData");
```
Comment (alelom) [inline InteroperabilityTest_Engine/Query/IsEqual.cs]:
```suggestion
          
```

File: InteroperabilityTest_Engine/Query/IsEqual.cs [hunk 2]
```diff
@@ -58,11 +58,9 @@ public static partial class Query
 
             comparer.Config.MembersToIgnore = config.PropertiesToIgnore;
 
-            if (config.IgnoreCustomData)
-                comparer.Config.MembersToIgnore.Add("CustomData");
+          
 
-            if (config.IgnoreGuid)
-                comparer.Config.TypesToIgnore.Add(typeof(Guid));
+            comparer.Config.TypesToIgnore.Add(typeof(Guid));
```
Comment (alelom) [inline InteroperabilityTest_Engine/Query/IsEqual.cs]:
```suggestion
           
```

File: InteroperabilityTest_Engine/Query/IsEqual.cs [hunk 3]
```diff
@@ -58,11 +58,9 @@ public static partial class Query
 
             comparer.Config.MembersToIgnore = config.PropertiesToIgnore;
 
-            if (config.IgnoreCustomData)
-                comparer.Config.MembersToIgnore.Add("CustomData");
+          
 
-            if (config.IgnoreGuid)
-                comparer.Config.TypesToIgnore.Add(typeof(Guid));
+
```
Comment (alelom) [inline InteroperabilityTest_Engine/Query/IsEqual.cs]:
```suggestion
```

File: InteroperabilityTest_Engine/Query/IsEqual.cs [hunk 4]
```diff
@@ -58,11 +58,8 @@ public static partial class Query
 
             comparer.Config.MembersToIgnore = config.PropertiesToIgnore;
 
-            if (config.IgnoreCustomData)
-                comparer.Config.MembersToIgnore.Add("CustomData");
+
```
Comment (alelom) [inline InteroperabilityTest_Engine/Query/IsEqual.cs]:
```suggestion
```

File: InteroperabilityTest_Engine/Query/IsEqual.cs [hunk 5]
```diff
@@ -99,13 +95,9 @@ public static partial class Query
             comparer.Config.MembersToIgnore = config.PropertiesToIgnore;
             comparer.Config.DoublePrecision = config.NumericTolerance;
 
-            if (config.IgnoreCustomData)
-            {
-                comparer.Config.MembersToIgnore.Add("CustomData");
-            }
+            comparer.Config.MembersToIgnore.Add("CustomData");
```
Comment (alelom) [inline InteroperabilityTest_Engine/Query/IsEqual.cs]:
```suggestion
```

File: InteroperabilityTest_Engine/Query/IsEqual.cs [hunk 6]
```diff
@@ -99,13 +95,8 @@ public static partial class Query
...
             comparer.Config.DoublePrecision = config.NumericTolerance;
 
-            if (config.IgnoreCustomData)
-            {
-                comparer.Config.MembersToIgnore.Add("CustomData");
-            }
 
-            if (config.IgnoreGuid)
-                comparer.Config.TypesToIgnore.Add(typeof(Guid));
+            comparer.Config.TypesToIgnore.Add(typeof(Guid));
```
Comment (alelom) [inline InteroperabilityTest_Engine/Query/IsEqual.cs]:
```suggestion
```

**Comment by al-fisher** [comment]
Super - thanks @FraserGreenroyd @IsakNaslundBh

**Pull request merged**